usecases/MinimumWorkingExample/dodo.py

```diff
@@ -18,9 +18,6 @@
 
```

Comment:
The Youngs modulus should be computed for all analyzed files in this setup and the results stored in the KG.

Reply:
I would suggest to use the "cheap" or "single file" option here as default, so that the CI tests are not taking forever, but implement a "full" mode to locally check that there are not problems with specific files.

Reply:
As a note, this is not actually outdated!!! The file has changed (for other reasons) but this point is still valid.